expyfun/stimuli/_crm.py

```diff
@@ -0,0 +1,523 @@
+"""Functions for using the Coordinate Response Measure (CRM) corpus.
+"""
+
+# Author: Ross Maddox <ross.maddox@rochester.edu>
+#
+# License: BSD (3-clause)
+
+import os
+from os.path import join
+import numpy as np
+from mne.filter import resample
+import expyfun.visual as vis
+from ._stimuli import window_edges
+from ..io import read_wav, write_wav
+from .._utils import fetch_data_file, _get_user_home_path
+from zipfile import ZipFile
+from expyfun._parallel import parallel_func
```

Comment:
all `expyfun` imports within the module should be relative, so `from ..parallel import parallel_func`